dallinger/heroku/messages.py

```diff
@@ -147,102 +154,225 @@ def get_email_server():
 """
 
 
-class EmailingHITMessager(object):
```

Comment:
Is it safe to remove this class? Might experimenters be using it directly? Perhaps we should re-implement this using the new and improved mechanism.

Reply:
It's still there. It just moved within the file so shows up as both a deletion and an addition.